blockchain/viewchange/mod.go

```diff
@@ -0,0 +1,23 @@
+package viewchange
+
+import (
+	"go.dedis.ch/fabric/blockchain"
+	"go.dedis.ch/fabric/mino"
+)
+
+// ViewChange provides primitives to verify if a participant is allowed to
+// propose a block as the leader. It is also responsible for verifying the
+// integrity of the players of the chain.
+type ViewChange interface {
+	// Wait returns a non-nil error when the node is allowed to make the
```

Comment:
- You return an error when you are allowed to make a proposal?
- I would use the same word in the comment and the code: in the comment you speak about "proposal" and in the code its "block". 
- It feels strange to suddenly speak about a "node". I would be a bit more precise by saying maybe "...returns a non-nil error when the node *running the code* is..."

Reply:
Oops, no it's a nil error when allowed. I changed node by player to be more consistent.